src/clang-cpp-frontend/clang_cpp_language.cpp

```diff
@@ -36,6 +38,9 @@ void clang_cpp_languaget::force_file_type()
   std::string clangstd = cppstd.empty() ? "-std=c++03" : "-std=c++" + cppstd;
   compiler_args.push_back(clangstd);
 
+  if(!config.options.get_bool_option("no-optimized-cpp-includes"))
+    compiler_args.push_back("-I" + esbmc_cpp_includes());
```

Comment:
In my opinion, this should depend on `--no-library` like the OM header path for C does, see <https://github.com/esbmc/esbmc/blob/db4d28d4bfdbb8a80f118c15ba6fa4f5f70ad0ea/src/c2goto/cprover_libc_sources.cpp#L86-L90>

As another comment: I don't think the `force_file_type()` method is the right place for placing standard include paths, though I also don't see a better one right now.

Reply:
I agree, I also didn't find a more suitable place.

Maybe we should change the function name to something else